content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Enabling Rate Limits using Envoy
+description: This task shows you how to use Istio to dynamically limit the traffic to a service.
+weight: 10
+keywords: [policies,quotas]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+This task shows you how to use Envoy's native rate limiting in Istio to dynamically limit the traffic to a
+service.
+
+## Before you begin
+
+1. Setup Istio in a Kubernetes cluster by following the instructions in the
+   [Installation Guide](/docs/setup/getting-started/).
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+    The Bookinfo sample deploys 3 versions of the `reviews` service:
+
+    * Version v1 doesn’t call the `ratings` service.
+    * Version v2 calls the `ratings` service, and displays each rating as 1 to 5 black stars.
+    * Version v3 calls the `ratings` service, and displays each rating as 1 to 5 red stars.
+
+    You need to set a default route to one of the versions. Otherwise, when you send requests to the `reviews` service, Istio routes requests to all available versions randomly, and sometimes the output contains star ratings and sometimes it doesn't.
+
+1. Set the default version for all services to v1.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
+    {{< /text >}}
```

Comment:
```suggestion

1. Set the default version for all services to v1 so that no ratings are returned (no stars are displayed).

    {{< text bash >}}
    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
    {{< /text >}}
```